758-disable slider when the browse results are empty

dstopp opened the pull request:
-remove spinner from slider and set html to "?"
-fix the width of the slider to the size of the "?" when empty
-disable the slider when browse results empty
-fix the bug where sometimes the cart message would appear
-change the slider number to a class instead of id for future use on the cart

dstopp committed:
-disable slider when the browse results are empty
-remove spinner from slider and set html to "?"
-fix the width of the slider to the size of the "?" when empty
-fix the bug where sometimes the cart message would appear
-change the slider number to a class instead of id for future use on the cart

rfrenchseti reviewed (changes requested):
As long as you're fixing problems with "empty results" showing up when it shouldn't, do you want to go ahead and fix the "browse" version above?

dstopp committed:
remove the empty message from the gallery when the gallery not empty.

dstopp committed:
-disable nav for both browse and cart
-move the message for empty cart into base.html
-minor class style change
-remove of unused class
-disable the table when the message is displayed

dstopp committed:
-on delete on cart tab, do not remove nav
-remove elements in cart on empty

dstopp committed:
use total_obs_count to determine empty results.

dstopp committed:
-don't try to retrieve on change of tab if the results are empty and nothing else has changed...

dstopp committed:
intermediate checkin, not yet fixed.

dstopp committed:
renamed class to be consistant w/standards

dstopp committed:
Merge branch 'new_ui' of https://github.com/SETI/pds-opus into 758_update_slider_on_empty

# Conflicts:
#	opus/application/static_media/js/browse.js

dstopp committed:
-moved some code around due to race conditions in emptyCart.

dstopp committed:
fixed merge issue on slider class

dstopp committed:
-use better way to empty gallery
-don't remove observations from cache one at a time
